src/components/calcite-value-list-item/calcite-value-list-item.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  Listen,
+  Method,
+  Prop,
+  h
+} from "@stencil/core";
+import { ICON_TYPES } from "../calcite-pick-list/resources";
+
+@Component({
+  tag: "calcite-value-list-item",
+  styleUrl: "./calcite-value-list-item.scss",
+  shadow: true
+})
+export class CalciteValueListItem {
+  // --------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  // --------------------------------------------------------------------------
+
+  @Prop() selected = false;
```

Comment:
sand: Should this be reflected?

Reply:
Good question. This component is basically acting as a pass-through to the pick-list-item, but that's in the shadow dom. I decided not to reflect it so I didn't have to add redundant logic to this component to update whenever the pick-list-item prop updated. Without that, this prop would be inaccurate after initial load after a selection changed.

Reply:
I think a user would expect this prop to be mutable and become true if the item is selected. Just like a checkbox. Should we wire that up?

Reply:
There's quite a bit of work to wire up to make this work in all use-cases. We'd need to add a lot of code to cover all scenarios and some of it would be redundant.

Mind holding off on this for now?

Reply:
Yep. enhancement issue would be good and we can tackle it later.